src/components/StatusDashboard/current_migrations.jsx

```diff
@@ -100,21 +102,32 @@ export default function CurrentMigrations({ onLoad }) {
             sort={state.sort.closed}
           />
         </table>
+        <table className={styles.migrations_table}>
+          <TableContent
+            collapsed={state.collapsed.paused}
+            name="Paused migrations"
+            resort={resort("paused")}
+            rows={paused}
+            select={() => select("paused")}
+            sort={state.sort.paused}
+          />
+        </table>
       </div>
     </div>
   );
 }
 
 function TableContent({ collapsed, name, resort, rows, select, sort }) {
   const [redirect, setState] = useState('');
+  console.log('name:', name);
   if (redirect) return (<Redirect to={redirect} replace={false} push={true} />);
   return (
     <>
       <thead>
         <tr onClick={select}>
           <th colSpan={8} className={collapsed ? styles.collapsed : undefined}>
             {name}{" "}
-            <span className="badge badge--secondary">{rows.length || "…"}</span>
+            <span className="badge badge--secondary">{name!=="Paused migrations" ? rows.length || "..." : rows.length || "0"}</span>
```

Comment:
Are we special casing "Paused migrations" here? I would expect the code to be something like:

- While fetching the remote JSON, the badge says `…` (note this is the three period symbol, not three periods one after another like `...`; this should be reverted).
- Once fetched, we calculate the `rows.length` and then report the number.

Is this too late in the code to check for rows being fetched? 🤔

Reply:
> Is this too late in the code to check for rows being fetched? 🤔

I don't know

Reply:
Some logics was added to address the mentioned point.